numpy/linalg/lapack_lite/lapack.f.patch

```diff
@@ -0,0 +1,56 @@
+@@ -57759,9 +57767,10 @@
+ *     Scan up each column tracking the last zero row seen.
+          ILACLR = 0
+          DO J = 1, N
++            I = M
+-            DO I = M, 1, -1
+-               IF( A(I, J).NE.ZERO ) EXIT
+-            END DO
++C           DO I = M, 1, -1
++C              IF( A(I, J).NE.ZERO ) EXIT
++C           END DO
```

Comment:
Well, if the other patch just works, then maybe that is easier. Anyway, let me know if there are issues; I'm old enough for most of my scientific code to have been written in fortran.